packages/clay-autocomplete/src/ClayAutocomplete.js

```diff
@@ -222,6 +252,8 @@ class ClayAutocomplete extends ClayComponent {
 	_handleItemSelected(event, index) {
 		const item = this.filteredItems[Number(index)];
 
+		this._defaultItemSelected();
```

Comment:
This call here is not necessary anymore since we have the listener for `itemSelected`.